### PR TITLE
Presets for new AOS Cine Frames

### PR DESCRIPTION
**Pull request by ChrisRosser**
Hi Everyone,

I'm launching some new Cine frames over the next few days and wanted to create presets so that people find them easy to tune. The tunes should be in the same style as previous submissions.

Thank you in advance for taking a look over them and I stand ready to make any alterations needed to esnure they meet our high Betaflight standards!

Also if you have any suggestions to improve the tunes based on your experience that would be incredible also.

### CODE REVIEW

File: presets/4.5/filters/aos_rc/aos_cine40_filters.txt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#$ TITLE: AOS Cine40 Filters
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: filter, filters, aos, chris, rosser, cine40
+#$ AUTHOR: Chris Rosser
+#$ DESCRIPTION: Developed for the AOS Cine40 based on a build with 2207 1980KV motors on 6S.
+#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATEMPT TO USE WITH OUT RPM FILTERING!
```
Comment (limonspb) [inline presets/4.5/filters/aos_rc/aos_cine40_filters.txt:8]: should be "WITHOUT" and in a few places

File: presets/4.5/tune/aos_rc/aos_cine20_tune.txt [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+set dyn_idle_p_gain = 40
+
+
+# ------ OPIONS GO BELOW THIS LINE ------
+
+# -- Filters --
+
+#$ OPTION_GROUP BEGIN: Filters
+    #$ OPTION BEGIN (UNCHECKED): AOS Filters (Recommended)
+    #$ INCLUDE: presets/4.5/filters/aos_rc/aos_cine20_filters.txt
```
Comment (limonspb) [inline presets/4.5/tune/aos_rc/aos_cine20_tune.txt:83]: need a tab for padding for whats inside OPTIONS

File: presets/4.5/filters/aos_rc/aos_cine20_filters.txt [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#$ TITLE: AOS Cine20 Filters
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: filter, filters, aos, chris, rosser, cine20
+#$ AUTHOR: Chris Rosser
+#$ DESCRIPTION: Developed for the AOS Cine20 based on a build with 1303 6000KV motors and 550mAh 4S battery.
+#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATEMPT TO USE WITH OUT RPM FILTERING!
```
Comment (haslinghuis) [inline presets/4.5/filters/aos_rc/aos_cine20_filters.txt:8]:
```suggestion
#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATTEMPT TO USE WITHOUT RPM FILTERING!
```

File: presets/4.5/filters/aos_rc/aos_cine30_filters.txt [hunk 4]
```diff
@@ -0,0 +1,37 @@
+#$ TITLE: AOS Cine30 Filters
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: filter, filters, aos, chris, rosser, cine30
+#$ AUTHOR: Chris Rosser
+#$ DESCRIPTION: Developed for the AOS Cine30 based on a build with 1507 3800KV motors on 4S.
+#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATEMPT TO USE WITH OUT RPM FILTERING!
```
Comment (haslinghuis) [inline presets/4.5/filters/aos_rc/aos_cine30_filters.txt:8]:
```suggestion
#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATTEMPT TO USE WITHOUT RPM FILTERING!
```

File: presets/4.5/filters/aos_rc/aos_cine35_filters.txt [hunk 5]
```diff
@@ -4,7 +4,7 @@
 #$ STATUS: COMMUNITY
 #$ KEYWORDS: filter, filters, aos, chris, rosser, cine35
 #$ AUTHOR: Chris Rosser
-#$ DESCRIPTION: Developed for the AOS Cine35
+#$ DESCRIPTION: Developed for the AOS Cine35 based on a build with 2104 1800KV motors on 6S
 #$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATEMPT TO USE WITH OUT RPM FILTERING!
```
Comment (haslinghuis) [inline presets/4.5/filters/aos_rc/aos_cine35_filters.txt:8]:
```suggestion
#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATTEMPT TO USE WITHOUT RPM FILTERING!
```

File: presets/4.5/filters/aos_rc/aos_cine40_filters.txt (hunk 1 above)
Comment (haslinghuis) [inline presets/4.5/filters/aos_rc/aos_cine40_filters.txt:8]:
```suggestion
#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATTEMPT TO USE WITHOUT RPM FILTERING!
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by limonspb** [comment]
@ChrisRosser Can't comment on the tune parameters much, but looks good overall.
Please fix the minor typos, and padding for options here and there.
New presets must be EXPERIMENTAL at first. But we are less strict about it now :)

Small problem:
if filter preset uses a slider or sliders, it's better to do `simplified_tuning apply`.

On a side note:
If you want, then filter options can be checked by default inside a tune preset.
Also sag compensation can be checked by default, since it goes under TUNE umbrella too.
And if you are using simplified tuning sliders, you don't have to list all pid values. Unless you want to.

**Comment by ChrisRosser** [comment]
@limonspb and @haslinghuis 

Thank you for your review and suggestions. I have made all the suggested whitespace changes. All of these tunes are based on the Cine25 and Cine35 tunes which have been used in the community for over a year without any issues so I'm confident they will be safe for anyone in the community to use. I've also set the MM to 1.0 on the new tunes which should be very very safe (I'm running around 1.5 MM on all my drones)👍

**Comment by limonspb** [comment]
> @limonspb and @haslinghuis
> 
> Thank you for your review and suggestions. I have made all the suggested whitespace changes. All of these tunes are based on the Cine25 and Cine35 tunes which have been used in the community for over a year without any issues so I'm confident they will be safe for anyone in the community to use. I've also set the MM to 1.0 on the new tunes which should be very very safe (I'm running 1.5 MM on all my drones)👍

Any reason you didn't use `simplified_tuning apply` in the filter settings?
This command sets the proper values for the active sliders only.

**Comment by ChrisRosser** [comment]
> > @limonspb and @haslinghuis
> > Thank you for your review and suggestions. I have made all the suggested whitespace changes. All of these tunes are based on the Cine25 and Cine35 tunes which have been used in the community for over a year without any issues so I'm confident they will be safe for anyone in the community to use. I've also set the MM to 1.0 on the new tunes which should be very very safe (I'm running 1.5 MM on all my drones)👍
> 
> Any reason you didn't use `simplified_tuning apply` in the filter settings? This command sets the proper values for the active sliders only

No reason, I'll add that line.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**